Cargo.toml

```diff
@@ -12,12 +12,13 @@ byteorder = "1.3.4"
 cid = "0.4.0"
 dag-cbor = { path = "dag-cbor" }
 dag-cbor-derive = { path = "dag-cbor-derive" }
-failure = "0.1.7"
+dag-pb = { path = "dag-pb", optional = true }
 futures = "0.3.4"
 libipld-base = { path = "base" }
 libipld-macro = { path = "macro" }
 multibase = "0.8.0"
 multihash = "0.10.1"
+thiserror = "1.0.11"
```

Comment:
I wonder if this could be "1.0" or is there something behind pinning a "past" 1.0 version?

Reply:
patch versions should be compatible. If the crates semver correctly, the best practice imo is to use the version that it was compiled against, like `cargo update` or `cargo upgrade` do.

Reply:
Oki, I'll resolve these then.